src/__demo__/PeriodDimension.stories.js

```diff
@@ -1,14 +1,26 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { storiesOf } from '@storybook/react'
 
 import PeriodDimension from '../components/PeriodDimension/PeriodDimension'
+import FixedPeriodSingleSelect from '../components/PeriodDimension/FixedPeriodSingleSelect'
 
 const selectedPeriods = [{ id: 'LAST_12_MONTHS', name: 'Last 12 months' }]
 
 storiesOf('PeriodDimension', module).add('None selected', () => {
     return <PeriodDimension onSelect={selected => console.log(selected)} />
 })
 
+storiesOf('PeriodDimension', module).add('FixedPeriodSingleSelect', () => {
```

Comment:
![image](https://user-images.githubusercontent.com/12590483/86352742-90c90280-bc66-11ea-9dbc-4caa35f78e9b.png)
I think there's a need to rename the other stories, as the names (see screenshot) don't indicate that 2 stories are for the period selector and one is for the new FixedPeriodSingleSelect. Alternatively, the FixedPeriodSingleSelect could be its own story module.

Reply:
If moved to a separate module, could you please add a case that shows the `allowedPeriodTypes` in action as well?